### ISSUE TITLE
fullActivities organisation and lrs_id

### ISSUE THREAD

**Issue by ht2** [LearningLocker/learninglocker#1036]
Current schema: 

```
{
  "organisationId": "xxx",
  "lrsId": "xxx",
  "id": "xxx"
}
```

Should be:

```
{
  "organisation": ObjectId("xxx"),
  "lrs_id": ObjectId("xxx"),
  "activityId": "xxx"
}

**Comment by ryansmith94** [comment]
Fixed in [xapi-statements v2.2.3](https://github.com/LearningLocker/xapi-statements/releases/tag/v2.2.3). I'll update the xapi-service soon to close this issue.

**Comment by ryansmith94** [comment]
Fixed in [xapi-service v1.2.5](https://github.com/LearningLocker/xapi-service/releases/tag/v1.2.5).
